app/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
+  "project_info": {
+    "project_number": "120966088760",
+    "firebase_url": "https://skygear-demo.firebaseio.com",
+    "project_id": "skygear-demo",
+    "storage_bucket": "skygear-demo.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:120966088760:android:c75c83ea93df60f9",
+        "android_client_info": {
+          "package_name": "io.skygear.chatdemo"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "120966088760-39kpetq6klqbk0mofkppnou3ij8dmq2n.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyDjG6XtzR9WiJLVgUk2W1cTgysBsiFMF7Q"
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 1
+        },
+        "appinvite_service": {
+          "status": 1,
+          "other_platform_oauth_client": []
+        },
+        "ads_service": {
+          "status": 2
+        }
+      }
+    }
+  ],
+  "configuration_version": "1"
+}
```

Comment:
I think we will not commit the `google-services.json` in public repo, please remove it.